hack/testing/init-log-stack

```diff
@@ -33,6 +33,7 @@ openshift_logging_image_prefix=$imageprefix
 openshift_logging_use_ops=${ENABLE_OPS_CLUSTER:-False}
 openshift_logging_fluentd_use_journal=${USE_JOURNAL:-}
 openshift_logging_fluentd_journal_read_from_head=${JOURNAL_READ_FROM_HEAD:-False}
+openshift_logging_es_log_appenders=['file','console']
```

Comment:
To clarify, this is only for testing. Hence me wondering if we need to set up the 'file' appender since I don't believe we have any tests that explicitly check the file-based logs of Elasticsearch.